job_offers_data_load/PostgreSQL_data_insert.py

```diff
@@ -0,0 +1,94 @@
+import pandas
+import sqlalchemy.orm
+import sqlalchemy.sql.schema
+from sqlalchemy import select
+import datetime
+from job_offers_data_load.PostgreSQL_tables_declaration import *
+
+
+def is_already_for_list_like(dataframe: pandas.DataFrame,
+                             iterator: int,
+                             dataframe_column_name: str,
+                             session: sqlalchemy.orm.session.Session,
+                             schema_from: sqlalchemy.sql.schema,
+                             schema_from_column) -> list:
+    """Returns list that will be used for creation of Table. Makes sure there will be no duplicates."""
+    temp_list = []
+    for item in dataframe.loc[iterator, dataframe_column_name]:
+        statement = select(schema_from).where(schema_from_column == item)
+        is_already = session.execute(statement).fetchone()
+        if is_already is None:
+            temp_list.append(schema_from(item))
+        else:
+            temp_list.append(is_already[0])
+    return temp_list
+
+
+def stamp_expired(data_df: pandas.DataFrame,
+                  session_maker: sqlalchemy.orm.session.sessionmaker) -> None:
+    """Checks if existing record in database is present in newly scraped data. If it isn't then it means it no longer exist
+    on site, therefore is expired."""
+    session = session_maker()
+    #  create a list of all job offer_url from DataFrame
+    data_df_urls = data_df['offer_url'].to_list()
+    #  get all existing offers from Database
+    existing_offers = session.query(JobOffer).all()
+
+    for offer in existing_offers:
+        if offer.offer_url not in data_df_urls:
+            offer.expired = 'true'
+            offer.expired_at = datetime.datetime.today()
+    session.commit()
+
+
+def update_tables(data_df: pandas.DataFrame,
```

Comment:
namings and documentation are again not clear. I just see a lot of general words: some dataframe, some Database, all_data???